Source/monster.cpp

```diff
@@ -1932,7 +1932,7 @@ void M2MStartHit(int mid, int i, int dam)
 #endif
 	}
 
-	if (i >= 0) // BUGFIX: Missing check for golems `&& i <= MAX_PLRS`
+	if (i >= 0) // BUGFIX: Missing check for golems `&& i < MAX_PLRS`

```

Comment:
What's funny is this function only ever gets called when Golems attack another monster. But if for example, a monster were to attack another monster (such as a mod maker adding a spell) it would flag that monster as having hit all monsters.

Reply:
It can in hellfire :)

You can baserk monsters.